test/api/routing.test.js

```diff
@@ -32,6 +33,18 @@ describe('Routing', () => {
 
   const routingResponse = { meta: {}, routes: {} };
 
+  const matrixMode = 'car';
+  // const matrixOrigin = [{
+  //   latitude: 40.70390, longitude: }];
+  const matrixOrigin = [{ 
+    latitude: 40.70390, 
+    longitude: -73.98690}];
```

Comment:
yeah something seems off with the formatting here /cc @corypisano

Reply:
looks like we need to configure lint and run in CI. (get on par with frontend setup). will add a shortcut ticket

Reply:
for now, can format like below so it matches `matrixDestination`
```
const matrixOrigin = [
  {latitude: 40.70390,  longitude: -73.98690}
];
```